app/controllers/TagManagementApi.scala

```diff
@@ -140,4 +140,11 @@ object TagManagementApi extends Controller with PanDomainAuthActions {
     Ok(Json.toJson(jobs))
   }
 
+  def getTagsAsXml() = APIAuthAction.async {
```

Comment:
We almost certainly don't want this endpoint to authenticated. The incopy processes that consume it will not be using panda (the current "security" mechanism that incopy uses is ip and user agent based).

It might be worth moving this endpoint to a new unsecured read only controller, separating out the public read api from the tag manager specific read write api.